tests/test_resource_tracker.py

```diff
@@ -284,3 +284,43 @@ def test_resource_tracker_sigterm(self):
     def test_resource_tracker_sigkill(self):
         # Uncatchable signal.
         self.check_resource_tracker_death(signal.SIGKILL, True)
+
+    @pytest.mark.skipif(sys.version_info < (3, 8),
+                        reason="SharedMemory introduced in Python 3.8")
+    def test_loky_process_inherit_multiprocessing_resource_tracker(self):
+        cmd = '''if 1:
+        from loky import get_reusable_executor
+        from multiprocessing.shared_memory import SharedMemory
+        from multiprocessing.resource_tracker import (
+            _resource_tracker as mp_resource_tracker
+        )
+
+        def rtracker_getattrs():
```

Comment:
To avoid confusion with the loky resource tracker, I would rename to something like:

```suggestion
        def mp_rtracker_getattrs():
```